### PR TITLE
sioyek: unstable-2022-08-30 -> 2.0.0

### PR DESCRIPTION
**Pull request by foo-dogsquared**
###### Description of changes

List of changelogs:

- https://ahrm.github.io/jekyll/update/2022/12/12/sioyek-2.html
- https://github.com/ahrm/sioyek/compare/8d0a63484334e2cb2b0571a07a3875e6ab6c8916..v2.0.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/sioyek/default.nix [hunk 1]
```diff
@@ -15,15 +15,15 @@
 , wrapQtAppsHook
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: rec {
```
Comment (sikmir) [inline pkgs/applications/misc/sioyek/default.nix:18]:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

File: pkgs/applications/misc/sioyek/default.nix [hunk 2]
```diff
@@ -15,15 +15,15 @@
...
   pname = "sioyek";
-  version = "unstable-2022-08-30";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "ahrm";
     repo = "sioyek";
-    rev = "8d0a63484334e2cb2b0571a07a3875e6ab6c8916";
-    sha256 = "sha256-29Wxg/VVQPDDzzxKcvMa1+rtiP4bDkPAB/JJsj+F+WQ=";
+    rev = "v${version}";
```
Comment (sikmir) [inline pkgs/applications/misc/sioyek/default.nix:25]:
```suggestion
    rev = "v${finalAttrs.version}";
```

File: pkgs/applications/misc/sioyek/default.nix [hunk 3]
```diff
@@ -74,7 +74,7 @@ stdenv.mkDerivation (finalAttrs: {
   meta = with lib; {
     homepage = "https://sioyek.info/";
     description = "A PDF viewer designed for research papers and technical books";
-    changelog = "https://github.com/ahrm/sioyek/releases/tag/v1.5.0";
+    changelog = "https://github.com/ahrm/sioyek/releases/tag/v${version}";
```
Comment (sikmir) [inline pkgs/applications/misc/sioyek/default.nix:77]:
```suggestion
    changelog = "https://github.com/ahrm/sioyek/releases/tag/v${finalAttrs.version}";
```

File: pkgs/applications/misc/sioyek/default.nix (hunk 2 above)
Comment (foo-dogsquared) [reply pkgs/applications/misc/sioyek/default.nix]: Done.

**Comment by fossimprove** [comment]
Im a casual nixos user, I see it has been 3 weeks on this PR, yet it is not merged by maintainer?
Does it require approval from @podocarp ?

@foo-dogsquared When can I expect it into unstable branch?

Im not using HM or flakes, so my way is straight configuration.nix.


Can this PR be force merged by another admin-level maintainer?

**Comment by foo-dogsquared** [comment]
> Im a casual nixos user, I see it has been 3 weeks on this PR, yet it is not merged by maintainer?

Only committers can merge commits. A package maintainer's responsibility is to ensure the package is working with the nixpkgs repository. After it is reviewed, it is good to be merge... that is if a committer saw the PR.

> Does it require approval from @podocarp ?

It is already reviewed with its issue resolved so at this point it isn't necessary. It is a matter of time at this point waiting for a committer to merge it.

> @foo-dogsquared When can I expect it into unstable branch?

Most likely immediately after merging the PR though you can [track it](https://nixpk.gs/pr-tracker.html?pr=206432) yourself to be sure.

**Comment by fossimprove** [comment]
@foo-dogsquared Thank you for explaining it well for me!

Cool to know more on Nix.

**Pull request merged**